src/java/picard/analysis/directed/TargetMetricsCollector.java

```diff
@@ -301,12 +380,77 @@ public void setBaitSetName(final String name) {
             this.metrics.PROBE_SET = name;
         }
 
+        private int getBasesAtMinimumQualityInBlock(final SAMRecord record, final AlignmentBlock block) {
```

Comment:
Two things: 1) I think the name for this could be clearer, since it's bases at or above minimum quality.  Perhaps getCountOfBasesPassingQualityThreshold()?  2) A one line javadoc would help too.